docs/running-on-yarn.md

```diff
@@ -913,7 +913,11 @@ with a mixed workload of applications running multiple Spark versions, since a g
 the shuffle service is not always compatible with other versions of Spark. YARN versions since 2.9.0
 support the ability to run shuffle services within an isolated classloader
 (see [YARN-4577](https://issues.apache.org/jira/browse/YARN-4577)), meaning multiple Spark versions
-can coexist within a single NodeManager. The
+can coexist within a single NodeManager. There is an issue
```

Comment:
Let's clean up the sentence a little:
"Note that versions of YARN prior to 3.3.2/3.4.0 have an issue[...] which requires setting `yarn.nodemanager.aux-services.<service-name>.system-classes` as a workaround. See issue for details."

Reply:
does this affect all 2.9.0 til 3.3.2?  it would be good to be explicit about the versions it affects.

Reply:
As I [mentioned above](https://github.com/apache/spark/pull/35223#pullrequestreview-859780638), this should only affect versions 3.3.0 and 3.3.1

Reply:
Thanks for comments, updated.